include/zephyr/dt-bindings/usb/audio.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_USB_AUDIO_H
+#define ZEPHYR_INCLUDE_DT_BINDINGS_USB_AUDIO_H
+
+/* USB Device Class Definition for Audio Devices Release 2.0 May 31, 2006
```

Comment:
nit: I'd suggest adding proper Doxygen docstrings here, and make all these values appear in a USB audio subgroup.

Reply:
I would like to use the descriptions from actual specification, but USB Device Class Definition for Terminal Types licensing has
```
...
A LICENSE IS HEREBY GRANTED TO REPRODUCE THIS SPECIFICATION FOR INTERNAL
USE ONLY. NO OTHER LICENSE, EXPRESS OR IMPLIED, BY ESTOPPEL OR OTHERWISE,
IS GRANTED OR INTENDED HEREBY.
...
```

Therefore I believe I cannot just copy the descriptions to doxygen tags. What would be the approach then?